resolutions/-resolution-annual-xxx.md

```diff
@@ -0,0 +1,32 @@
+# ANNUAL RESOLUTIONS OF THE BOARD OF DIRECTORS
+
+**Resolution(s) in writing** of the board of directors of Hypha Worker Co-operative Inc. (the "Co-operative") 
+from the meeting of the directors held by electronic means on this [XXth day of Month, 20XX].
+
+**BACKGROUND:**
+
+1.  ...
+
+
+**IT WAS RESOLVED THAT:**
+
+1.  ...
+
+2.  ...
+
+
+Dated in the Province of Ontario on the [XXth day of Month, 20XX].
+
+__________________________________ :lock_with_ink_pen: [digital signature][sigfile-[director]]\
+[Director Name]
+
+__________________________________ :lock_with_ink_pen: [digital signature][sigfile-[director]]\
+[Director Name]
+
+__________________________________ :lock_with_ink_pen: [digital signature][sigfile-[director]]\
+[Director Name]
+
+
+<!-- Links -->
+   [sigfile-[director]]: resolution-XXX.md.[director].asc
+
```

Comment:
3-space indent is weird. I think we should have two lines indicating `director-1` and `director-2` so it's clear. This whole linking structure is confusing enough.


```suggestion
    [sigfile-[director-1]]: resolution-XXX.md.[director-1].asc
    [sigfile-[director-2]]: resolution-XXX.md.[director-2].asc

```

Reply:
I'm just copying over the structure from the other template -- I agree that I'm not really a fan of this link structure but happy to work with it